samples/batch/python/README.md

```diff
@@ -1 +1,15 @@
-Examples in C# and Python on how to use the Speech Services Batch Transcription API.
+Example in Python on how to use the Speech Services Batch Transcription API.
+
+To execute the sample you need to generate the Python client library.
+
+To do this follow these steps:
+
+1. Go to https://editor.swagger.io
+2. Click **File**, then click **Import URL**
+3. Enter the Swagger URL including the region for your Speech Services subscription: https://<your-region>.cris.ai/docs/v2.0/swagger
```

Comment:
The `<your-region>` isn't show in the markdown.

Reply:
addressed